### DISCUSSION TITLE
Make Tina save frontmatter in TOML

### DISCUSSION THREAD

**Discussion by ultimatedrivingtours** [tinacms/tinacms#3530] [Help] [answered]
I'm porting a Hugo website from Forestry to Tina. With a bit of experimentation and [this article](https://tina.io/blog/tina-v-1.1.3) I got Tina to recognize the existing toml frontmatter. 

```javascript
  schema: {
    frontmatterFormat: 'toml',
    frontmatterDelimiters: ['string', 'string'],
    collections: [
      {
        label: 'house',
```

However, when I create a new file under my collections, it saves in yaml. Is there a setting for that?

**Comment by jamespohalloran** [comment]
Is it saving with yml delimiters? This line should be:
```diff
-    frontmatterDelimiters: ['string', 'string'],
+.  frontmatterDelimiters: '+++' // OR [ '+++' , '+++' ] would also work
```

**Comment by logan-anderson** [comment, marked as answer]
Hello @ultimatedrivingtours `[string, string]` is the type of `frontmatterDelimiters`. Also, this is configured at the collection level. This allows you to configure how the frontmatter is parsed on a per collection basis. 

For example.

```ts
schema: {t
  collections: [ 
        {
            name: 'house',
            frontmatterFormat: 'toml',
            frontmatterDelimiters: '+++',
            //...
```

[check out the collection docs](https://tina.io/docs/reference/collections/) for more information.

**Reply by ultimatedrivingtours** [reply]
Thanks @logan-anderson
